ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -1795,6 +1790,20 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
           const somaticAssentAddendum = consentAddendumPediatric?.questionsAnswers
             ?.find(({answer, stableId}) => stableId === 'SOMATIC_ASSENT_ADDENDUM' && answer);
 
+          const consentAddendum = this.participant.data.activities
+            ?.find(({activityCode}) => activityCode === 'CONSENT_ADDENDUM');
+
+          let somaticConsentAddendumTumorAdult;
+
+          if (this.participant.data.ddp?.toLowerCase() == 'cmi-lms') {
```

Comment:
Worth noting: this is an active decision to look at the study name from ddp instead of the selected realm because this decision is based on a value in this particular index.